examples/lstm_conv.py

```diff
@@ -0,0 +1,136 @@
+from keras.models import Sequential
```

Comment:
I just decribed what this example is supposed to do.
For the accuracy and time per epoch, temporaly I don't have access to any gpu.
So I cannot really train it.
I renamed the file to conv_lstm